azkaban-web-server/src/main/java/azkaban/webapp/Status.java

```diff
@@ -29,18 +29,21 @@
 public class Status {
 
   private final String version;
+  private final String pid;
   private final String installationPath;
   private final long usedMemory, xmx;
   private final boolean isDatabaseUp;
   private final Map<Integer, Executor> executorStatusMap;
 
   Status(final String version,
+      final String pid,
```

Comment:
@kunkun-tang 
Yes. This is the pid for the web server. It is not about 'monitoring' the pid. This API is about reporting the current status of the application and pid is an important piece of information here. I agree the executor piece is more important and we do have plans to add that as well.